core/src/main/kotlin/com/strumenta/kolasu/transformation/Transformation.kt

```diff
@@ -578,7 +607,7 @@ open class ASTTransformer(
         return nodeFactory
     }
 
-    fun <T : Node> registerIdentityTransformation(nodeClass: KClass<T>) =
+    inline fun <reified T : Node> registerIdentityTransformation(nodeClass: KClass<T>) =
```

Comment:
What's the reason for inline+reified here?

Reply:
That is a good question. I think I needed to access the node type in some method invoked from here and the IDE suggested this refactoring.

In particular I think it was due to this other change:
```
inline fun <S : Any, reified T : Node> registerNodeFactory(
```
this became inlines and T parameter became reified, so that there we could write: `val node = T::class.dummyInstance()`. But given `registerIdentityTransformation` call `registerNodeFactory`, that had to be changed too. Perhaps I can add a comment about that.

Reply:
Ok, thanks.